js/index.js

```diff
@@ -0,0 +1,36 @@
+// Psuedo code
+// 1 - doc ready
+// 2 - make submit clickable via .submit
+// 3 - read the value entered into the field; store as var
+// 4 - make change all vals to lowercase
+// 5 - if-else to say if var is city name, then  render new background image
+
+
+$(document).ready(function() {
+	//console.log('got here');
+
+	$('#submit-btn').click(submit);
+
+
+	function submit (event) {
+		event.preventDefault();
+		var location = $('#city-type').val();
+		var location = location.toLowerCase().trim();
+		$('#city-type').val('');
+	
+
+
+	if (location === 'nyc' || location === 'new york' || location === 'new york city') {
+		$('body').toggleClass('nyc');
```

Comment:
You got the right idea here. The problem is two fold:

1. `toggleClass()` adds or removes whatever class you pass it. So in this case, you're saying, if they type 'nyc' once, add the class 'nyc' to the `body` tag. If they type in 'nyc' (or 'new york city', etc) again, remove the class 'nyc'. Whenever someone types in New York, we always want to display the new york background image via the 'nyc' class.

2. The second problem is that if the use types 'SF' and then we add the class 'sf' to the `body` tag - everything is fine. But then the user types in 'new york' and then we add the 'nyc' class to the body. The problem is, we didn't remove any of the other city class names that may have been added previously.

What you should do before adding the appropriate class based on what the user typed, you should first do some clean up on the `body` tag. This could look something like this:

```js
// Remove all the cities 
$('body').removeClass('nyc la sf austin sydney');

if (location === 'nyc' || location === 'new york' || location === 'new york city') {
   // now that we've removed any classes, we can now ADD the one we want
   $('body').addClass('nyc');
} // ...
```